src/components/Status/Status.props.ts

```diff
@@ -0,0 +1,12 @@
+import { StatusConfig } from './Status.styles';
+import { StatusAppearance } from './StatusAppearance.type';
+import { StatusEmphasis } from './StatusEmphasis.type';
+
+type Status = string;
```

Comment:
I don't think there is any need to add this abstraction. It would be better to add `label?: string` at line 11

Reply:
you're right - changed